### PR TITLE
add a way to upload binary file

### PR DESCRIPTION
**Pull request by Winultimatum**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add a parameter that takes in imput base64 to upload binary file to XSOAR.
Base64 encoding is just a way to pass binary as a parameter, the function will decode the base64 and insert the output in the file.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/XSOARFileManagement/Integrations/XSOARFileManagement/XSOARFileManagement.py [hunk 1]
```diff
@@ -339,32 +368,49 @@ def upload_file_command(client: Client, args: dict) -> CommandResults:
...
+                                      file_name,
+                                      target == 'incident attachment')
     else:
         arg_path: str = list(filter(None, [entry_id, file_path]))[0]
         res_path, res_name = get_file_path_name(arg_path)
         # file name override by user
         file_name = file_name if file_name else res_name
         if not file_name:
-            return_error("Impossible to detect a filename in the path, use the argument 'fileName' to set one !")
+            return_error("Impossible to detect a filename in the path,"
```
Comment (MosheEichler) [inline Packs/XSOARFileManagement/Integrations/XSOARFileManagement/XSOARFileManagement.py:406]:
```suggestion
            return_error("Impossible to detect a filename in the path, "
```

File: Packs/XSOARFileManagement/Integrations/XSOARFileManagement/XSOARFileManagement.py [hunk 2]
```diff
@@ -339,32 +368,49 @@ def upload_file_command(client: Client, args: dict) -> CommandResults:
...
+                         "use the argument 'fileName' to set one !")
         file_binary = open(res_path, 'rb')
-        response = client.upload_file(incident_id, file_binary, file_name, target == 'incident attachment')
+        response = client.upload_file(incident_id,
+                                      file_binary,
+                                      file_name,
+                                      target == 'incident attachment')
         file_binary.close()
-    demisto.results(response)
```
Comment (MosheEichler) [inline Packs/XSOARFileManagement/Integrations/XSOARFileManagement/XSOARFileManagement.py:367]: Why did you remove it?

**Review by MosheEichler — COMMENTED** [review]
Hi @Winultimatum,
Thank you for your contribution!Good work :)

Plesae see my comments,
We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.  

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

File: Packs/XSOARFileManagement/Integrations/XSOARFileManagement/XSOARFileManagement.py (hunk 2 above)
Comment (Winultimatum) [reply Packs/XSOARFileManagement/Integrations/XSOARFileManagement/XSOARFileManagement.py:367]: It outputs the result of the API and it's not useful, I missed it when I published the integration ^^"
